Northstar.Custom/mod/scripts/vscripts/sh_northstar_custom_precache.gnut

```diff
@@ -7,7 +6,10 @@ void function NorthstarCustomPrecache()
 	PrecacheWeapon( "mp_titanweapon_triplethreat" )
 	PrecacheWeapon( "melee_pilot_kunai" )
 
-	// create kunai damage source so game won't crash when we hit smth with it
-	// just using the default melee one, easier than making a new one
-	getconsttable()[ "eDamageSourceId" ][ "melee_pilot_kunai" ] <- eDamageSourceId.melee_pilot_emptyhanded
+	RegisterWeaponDamageSources(
+		{
+			mp_weapon_peacekraber = "Peacekraber",
+			melee_pilot_kunai = "Kunai"
```

Comment:
i would prefer if you would use localisation strings here, peacekraber should have one, idk about kunai

Reply:
Ah, I thought about that but forgot! I'll add it in

Reply:
On testing, apparently neither the peacekraber nor kunai have any localization strings. Should I use localization strings and add in only the English ones for now?

Reply:
Might not be worth given the plans to replace the peacekraber with the peacekeeper? Your call